src/plugins/vis_type_timeseries/server/plugin.ts

```diff
@@ -115,6 +120,13 @@ export class VisTypeTimeseriesPlugin implements Plugin<VisTypeTimeseriesSetup> {
       searchStrategyRegistry,
     };
 
+    setDataSourceEnabled({ enabled: !!plugins.dataSource });
+    core.savedObjects.addClientWrapper(
+      11,
```

Comment:
Hi Huy, how we we decide this priority here.

Reply:
The lower the number, the higher the priority a client wrapper is. After some investigation, these should be unique and since `10` was used for Vega, 11 was chosen.